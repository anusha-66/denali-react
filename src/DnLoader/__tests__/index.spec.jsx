/**
 * Copyright 2020, Verizon Media
 * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
 */

import React from 'react';
import { render } from 'enzyme';
import { DnLoader } from '../../index'; // src/index.tsx

describe('DnLoader Tests', () => {
    it('should render with default props', () => {
        expect.assertions(1);

        const rendered = render(<DnLoader />);

        expect(rendered).toMatchSnapshot();
    });

    it('should render correctly with custom props', () => {
        expect.assertions(1);

        const rendered = render(<DnLoader isInverse className="p-18" />);

        expect(rendered).toMatchSnapshot();
    });
});
