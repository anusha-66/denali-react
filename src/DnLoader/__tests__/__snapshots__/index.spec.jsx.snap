// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DnLoader Tests should render correctly with custom props 1`] = `
<div
  class="loader loader--inverse p-18"
/>
`;

exports[`DnLoader Tests should render with default props 1`] = `
<div
  class="loader"
/>
`;
